mgmtsystem_hazard/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Hazard & Risks
+==============
+
+This module enables you to manage the hazards and risks of your health and safety management system.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* go to Settings > Companies > Companies and select your company
+* in the configuration tab, select the risk computation formulae
```

Comment:
I cannot find the risk computation formulae

Reply:
I didn't notice, but that's because the field has been moved to module hazard_risk since V8 actually. I'll remove that from here.